src/App.js

```diff
@@ -5,10 +5,9 @@ import CreateList from './pages/CreateList/CreateList';
 
 const App = () => {
   return (
-    <main>
-      <h1>Welcome to our Smart Shopping List!</h1>
+    <main className="background-style">
```

Comment:
Relacionado con este [mensaje](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/pull/41/files#r820130945). Por favor, una mirada 👀

Reply:
Solucionado.

Reply:
listo ***c6231c109871a2ca93e66bfa807f4bb0e7f23644***